src/training_hub/algorithms/gepa.py

```
@@ -0,0 +1,469 @@
+        # Save best candidate prompt
+        best = result.best_candidate
+        with open(os.path.join(output_dir, "best_candidate.json"), "w") as f:
+            json.dump(best, f, indent=2)
+
+        # Save full result dict
+        try:
+            result_dict = result.to_dict()
+            with open(os.path.join(output_dir, "result.json"), "w") as f:
+                json.dump(result_dict, f, indent=2, default=str)
+        except Exception:
+            pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)